web/templates/tupa/forms/maksimi_suoritus.html

```diff
@@ -10,13 +10,15 @@
 Parhaat pisteet saa:<br>
 <input type="radio" id="{{parhaan_haku.id}}" value="min" name="{{parhaan_haku.name}}"
 {% ifequal parhaan_haku.value "min" %}checked="checked" {% endifequal %}
-/> pienin</label><br>
```

Comment:
Tässä ja vastaavissa vois käyttää `<label for="{{parhaan_haku.id}}">` tai wräpätä input labelin sisälle. Ei tarvis kliksutellessa osua radiopalloon :)

Reply:
Tein tohon nyt korjauksen kun noilla inputeillahan oli samat id:t, mikä ei ole ok.

Reply:
En lähtenyt tuota samaa kuitenkaan korjaamaan muihin vastaaviin kun tälläiset korjaukset ei ehkä sovi tän PR:n sisälle. Ja on sitten kivampi korjailla tälläsiä kun nämä templaatit on ensin edes jotenkin formatoitu.